Source/DataStructureAndEncodingDefinition/gdcmAttribute.h

```diff
@@ -79,14 +79,14 @@ template<> class VRVLSize<1> {
  * Attribute<0x0018,0x1182, VR::UL, VM::VM2> fd3 = {0,1}; // UL is not valid VR
  */
 template<uint16_t Group, uint16_t Element,
-   long int TVR = TagToType<Group, Element>::VRType, // can the user override this value ?
+   long long TVR = TagToType<Group, Element>::VRType, // can the user override this value ?
    int TVM = TagToType<Group, Element>::VMType // can the user override this value ?
    /*typename SQAttribute = void_*/ > // if only I had variadic template...
 class Attribute
 {
 public:
   typedef typename VRToType<TVR>::Type ArrayType;
-  enum { VMType = VMToLength<TVM>::Length };
+  enum:long long { VMType = VMToLength<TVM>::Length };
```

Comment:
Why does this patch touches `VM`, I assumed this should only touches `VR` stuff

Reply:
`enum:long long { VMType = VMToLength<TVM>::Length };`
is probably not necessary here (2 places), not sure

Reply:
@malaterre 
should i remove it? will probably work without, IMHO. But can not make Windows tests just now.

Reply:
@malaterre
Done revert VMType change